client/cl_property.lua

```diff
@@ -160,7 +160,7 @@ end
 function Property:RegisterGarageZone()
     if not next(self.propertyData.garage_data) then return end
 
-    if not (self.has_access or self.owner) or not self.owner then
+    if not (self.has_access or self.owner) then
```

Comment:
Can we make this into a config? 

-- If you set it to `true`, anyone you allow access to your property will also be able to access your garage.
Config.AllowAccessToGarage

And set to false by default, just in case

Reply:
Done <3